txn_writer.go

```diff
@@ -19,6 +19,18 @@ import (
 	"github.com/sdboyer/gps"
 )
 
+// Example string to be written to the manifest file
+// if no dependencies are found in the project
+// during `dep init`
+const exampleToml = `
+# Example:
+# [[dependencies]]
+# branch = "master"
+# name = "github.com/vendor/package"
+# revision = "abc123"
```

Comment:
@EwanValentine "revision" is sort of intentionally underspecified; its valid values vary depending on the underlying source type. For git and hg, it's a SHA1; for bzr, it's this [weird 3-part internal revid](http://doc.bazaar.canonical.com/bzr.1.13/en/user-guide/#specifying-revisions); for svn, it is (I guess, we actually don't really support this yet) the global svn revision number.

IMO, it's for completeness and clarity, it's probably worth putting in this information (feel free to wordsmith 😛) right there in the comment block, so that users have it immediately on hand. It's at least worth doing that until we have more proper central documentation out there...